assets/js/ledyer-checkout-for-woocommerce.js

```diff
@@ -443,24 +435,15 @@ jQuery(function ($) {
 
             $(document).on('ledyerCheckoutOrderComplete', function (event) {
                 lco_wc.logToFile('ledyerCheckoutOrderComplete from Ledyer triggered');
-
-                if (lco_wc.redirectUrl !== null) {
-                    // This means that placeLedyerOrder was called successfully already
-                    // (Due to an earlier call caused by client validation)
-                    lco_wc.redirectUrl.searchParams.append('lco_purchase_complete', 'yes');
-                    window.location.href = lco_wc.redirectUrl.toString();
-                    return;
-                }
```

Comment:
This was put in place in order to avoid duplicate orders (due to placeLedyerOrder being called twice), make sure to verify this in HK staging

Reply:
Otherwise we will go through placeLedyerOrder one more time on the next submit, after a failed submit and redirect from there